### PR TITLE
[doc] Add metering HLD

### PR DESCRIPTION
**Pull request by r12f**

### CODE REVIEW

File: documentation/metering/metering.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+| SAI_METER_POLICY_ATTR_IP_ADDR_FAMILY | `sai_ip_addr_family_t` | SAI_IP_ADDR_FAMILY_IPV4 | IP address family of the metering policy |
+
+#### 3.3.3. Meter rule
+
+| Attribute | Type | Default Value | Description |
+| --- | --- | --- | --- |
+| SAI_METER_RULE_ATTR_METER_POLICY_ID | `sai_object_id_t` | SAI_NULL_OBJECT_ID | Meter policy ID of this meter rule. |
+| SAI_METER_RULE_ATTR_METER_CLASS | `sai_uint32_t` | UINT32_MAX | Meter class when this meter rule is hit. |
+| SAI_METER_RULE_ATTR_DIP | `sai_ip_address_t` | NA | Destination IP for ternary match. |
+| SAI_METER_RULE_ATTR_DIP_MASK | `sai_ip_address_t` | NA | Destination IP mask for ternary match. |
```
Comment (marian-pritsak) [inline documentation/metering/metering.md:126]: Any reason why it's ternary and not LPM?

Comment (r12f) [reply documentation/metering/metering.md:126]: this is because priority is being used for crafting the metering rules.

Comment (marian-pritsak) [reply documentation/metering/metering.md:126]: But are they prefixes or arbitrary bits?

Comment (r12f) [reply documentation/metering/metering.md:126]: they are indeed prefixes, not arbitrary bits.

**Comment by r12f** [comment]
Confirmed with Sushant (in reviewer list) offline and merging the HLD.

**Pull request merged**